cha/sota/test/procedure.tex

```diff
@@ -0,0 +1,36 @@
+\subsection{Procedure}\label{subsec:procedure}
+
+Referring back to \jjwarning{Ref to the list from the whywetest.tex}, the
+first aspect we wanted to test was offset between playing devices.
```

Comment:
Jeg synes at alle fire forskellige aspekter skal nævnes her, som en form for indledning til subsektionen.